mautic/mautic
# Multiple companies management

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | https://github.com/mautic/mautic-documentation/pull/294
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

We need use only one company to each contact. This PR added option to not use multiple companies to contact



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. We add new option to configuration 
![image](https://user-images.githubusercontent.com/462477/151988239-c4db4996-ef01-42f7-ae52-52b5150721b5.png)

By default it is enabled and should works like before. You can use companies to contact same like before.

3. If you disable it, you can use  just one company to assign to contact

![image](https://user-images.githubusercontent.com/462477/151988489-23df9f8a-699b-416a-8039-d3a05f6a2f22.png)

4. You can test any combination
5. Also test  new command `mautic:contact:delete:secondary-companies` to delete all secondary companies, If you migrate to  to single company managment.






<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10817"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
